.github/workflows/notify-doc.yml

```diff
@@ -0,0 +1,23 @@
+name: notifier for docteam
+
+on:
+    pull_request_target:
+        types: [ labeled, closed ]
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout repo content
+              uses: actions/checkout@v2
+
+            - name: Setup python
+              uses: actions/setup-python@v2
+              with:
+                  python-version: 3.9
+
+            - name: Install python packages
+              run: pip install -r scripts/requirements.txt
+
+            - name: Analyze the event and (if needed) write a comment
+              run: python scripts/with_pygithub/notifier.py --token ${{ secrets.WORKFLOW_GITHUB_TOKEN }} --event ${{ github.event.action }} --pull_request ${{ github.event.pull_request.number }} --label "${{ github.event.label.name }}" --repo ${{ github.repository }}
```

Comment:
I don't think you need to pass the label to the notifier here. See my proposal for `if` simplification.